<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QMainWindow" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Reader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Users/37544/Downloads/Folder of Images/gallery (1).png</normaloff>../../Users/37544/Downloads/Folder of Images/gallery (1).png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 216, 255);
border-color: rgb(147, 143, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>971</width>
      <height>941</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Image Reader</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ProviderButton">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Users/37544/Downloads/folder.png</normaloff>../../Users/37544/Downloads/folder.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OpenFileButton">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Users/37544/Downloads/Folder of Images/gallery.png</normaloff>../../Users/37544/Downloads/Folder of Images/gallery.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="listView">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>400</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(249, 216, 255);
background-color: rgb(220, 217, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(189, 176, 255), 2px;</string>
         </property>
         <property name="title">
          <string>Image Info</string>
         </property>
         <widget class="QLabel" name="FileName">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>File Name:</string>
          </property>
         </widget>
         <widget class="QLabel" name="SizeLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>221</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
         <widget class="QLabel" name="Resolution">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>221</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Resolution:</string>
          </property>
         </widget>
         <widget class="QLabel" name="ColorDepth">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>221</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Color depth:</string>
          </property>
         </widget>
         <widget class="QLabel" name="Compression">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>220</y>
            <width>221</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Compression:</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
